src/web/components/elements/InstagramBlockComponent.tsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
 
-import { unescapeData } from '@root/src/lib/escapeData';
 import { css } from 'emotion';
 
-const widthOverride = css`
-	iframe {
-		/* The instagram embed js hijacks the iframe and calculated an incorrect width, which pushed the body out */
```

Comment:
This comment suggests we might need to handle the instagram js doing something to override our width styles

Reply:
I think this is talking about the iframe that the instagram js creates its-self, Im not sure this is a problem with these changes.